homeassistant/components/overkiz/sensor.py

```diff
@@ -484,6 +484,10 @@ def native_value(self) -> StateType:
         if (
             state is None
             or state.value is None
+            # It seems that in some cases we return `None` if state.value is falsy.
```

Comment:
I am not sure what is wrong here? Return `None` will set the sensor to unknown. This is the state we want to express, when Overkiz doesn't provide a state at the moment.

Reply:
Wit the current code e.g. it will still return `None` if `state.value` is `0` or `0.0` and the state class is != `MEASUREMENT`. That is the case that is probably incorrect here.

Reply:
Why are we not fixing the issue instead of adding this note?

Reply:
I fixed the issue. After that Martin requested to add comment. Feel free to suggest a fix. I cannot test this.

Reply:
Me neither but I now checked the previous pr for context so I understand this. 
But @iMicknl maybe can test?

Reply:
Yes, let add it. When someone has time to look at this and fix it they can remove the comment.